release notes/v0.8.0.md

```diff
@@ -1,43 +1,44 @@
 xk6-browser v0.8.0 is here! :tada:
 
-We've been busy preparing for a big change which will occur very soon :crossed_fingers:, so this minor release is smaller than usual. It contains fixes, refactoring of code, and an exciting new addition... `async`/`await` keyword support, so no more messy `promise` `.then` chains!
+This release contains preparation for the upcoming `xk6-browser` and `k6` merge, preparations for the upcoming Async API features, bug fixes, and code refactoring.
 
 
 ## Experimental Module Merge into k6
 
-We have refactored code in preparation for the merge into `k6`. This will mean that all future releases of `xk6-browser` will also be part of the `k6` releases (as an experimental module). Merging it into `k6` will be super useful for you and us as it will increase the reach and visibility of the tool, which should mean more feedback and guidance from the community to allow us to tailor the product to your needs. It's worth noting that we're not merging the two code bases into a single one.
+We've been busy preparing for this big change that will occur very soon :crossed_fingers:. We have [refactored the code](https://github.com/grafana/xk6-browser/issues/683) in preparation for the merge into `k6` (see [the related issue list](https://github.com/grafana/xk6-browser/issues?q=label%3Ak6-core-compatibility+is%3Aclosed)). This will mean that all future releases of `xk6-browser` will also be part of the `k6` releases (as an experimental module). Merging `xk6-browser` into `k6` will allow you to run both the protocol and browser tests using the same `k6` command line tool. Doing so will increase the reach and visibility of `xk6-browser` and allow us to collect more feedback and guidance from the community to tailor our product to your needs. _It's worth noting that we're not merging the two code bases into a single one. `xk6-browser` will still be in its own repository._
```

Comment:
```suggestion
We've been busy preparing for this big change that will occur very soon :crossed_fingers:. We have [refactored the code](https://github.com/grafana/xk6-browser/issues/683) in preparation for the merge into `k6` (see [the related issue list](https://github.com/grafana/xk6-browser/issues?q=label%3Ak6-core-compatibility+is%3Aclosed)). This will mean that all future releases of `xk6-browser` will also be part of the `k6` releases (as an experimental module). Merging `xk6-browser` into `k6` will allow you to run both the protocol and browser tests using the same `k6` command line tool. This will also increase the reach and visibility of `xk6-browser` and allow us to collect more feedback and guidance from the community to tailor our product to your needs. _It's worth noting that we're not merging the two code bases into a single one. `xk6-browser` will still be in its own repository._
```

Reply:
👍